satip/utils.py

```diff
@@ -845,8 +847,17 @@ def save_to_zarr_to_s3(dataset: xr.Dataset, filename: str):
         log.debug(f"New times for {path}: {new_times}", memory=get_memory())
 
         log.debug(f"Saved to temporary file {path}, now pushing to {filename}", memory=get_memory())
-        # save to s3
-        filesystem = fsspec.open(filename).fs
+
+        # Save to the specified backend
+        filesystem = fsspec.open(
+            filename,
+            target_options={
+                "storage_options": {
+                    "token": "env"
```

Comment:
Could we change this to use an environment variable? Maybe `STORAGE_BACKEND_TOKEN`? So then it can be easily set programmatically?

Reply:
> Could we change this to use an environment variable? Maybe `STORAGE_BACKEND_TOKEN`? So then it can be easily set programmatically?

I've added the required changes in my latest commit

Reply:
> > Could we change this to use an environment variable? Maybe `STORAGE_BACKEND_TOKEN`? So then it can be easily set programmatically?
> 
> I've added the required changes in my latest commit

Hi @jacobbieker , do the new commits contain what was expected?

Reply:
Thanks for the changes! Sorry for the delayed reply, I was off the last few days last week. I've added a newer review. This was close to what I was thinking!